cmd/swarm/swarm-smoke/upload_and_sync.go

```diff
@@ -65,6 +65,14 @@ func uploadAndSyncCmd(ctx *cli.Context, tuid string) error {
 
 		return e
 	}
+
+	// trigger debug functionality on randomBytes even on successful runs
+	err := trackChunks(randomBytes[:])
+	if err != nil {
+		log.Error(err.Error())
```

Comment:
I wonder if we shouldn't return an error in this case. I know it's an edge case, as the actual test passed, but something else went wrong - unlikely but still possible, and it would show up as success in grafana. WDYT?

Reply:
We care mostly about the result of the smoke test. If for some reason the `debug` calls after that fail, we should not log the test as a failure, but still notice the error.